jupyterhub/templates/hub/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           persistentVolumeClaim:
             claimName: hub-db-dir
         {{- end }}
-      {{- if .Values.rbac.enabled }}
+      {{- if .Values.rbac.create }}
       serviceAccountName: hub
```

Comment:
Maybe this can be user specified instead, like in https://github.com/helm/charts/blob/5cd842068b06fcb41dd70085af5b070c07b4c671/stable/nginx-ingress/templates/controller-deployment.yaml#L240

This might be an extra breaking change, because I think the current of setting `.Values.rbac.enabled` to `false` is to basically use the default service account